Debug/Editor/Guides.cs

```diff
@@ -0,0 +1,99 @@
+using UnityEditor;
```

Comment:
To reviewers - this is only available in dev checkouts, it is not included in the built package. It's really handy when debugging pivot issues, so I added to the repo.